godotproject/DeathPlane.gd

```diff
@@ -14,4 +16,4 @@ func _on_DeathPlane_body_entered(body):
 			Global.fade_set_body_to_position(body, level1_start_postion)
```

Comment:
```suggestion
			Global.fade_set_body_to_position(body, start_position)
```
We should only need one `start_position` variable and remove the `if` statement here